mitosheet/mitosheet/tests/test_transpile.py

```diff
@@ -550,6 +550,75 @@ def test_transpile_as_function_single_param_multiple_times(tmp_path):
         f"txt, txt_1 = function(var_name)"
     ]
 
+@pandas_post_1_2_only
+@python_post_3_6_only
+def test_transpile_fully_parameterized_function_string(tmp_path):
```

Comment:
This test is almost perf - I think we just need to pass a dataframe + df_names to the `create_mito_wrapper` - to test the dataframe parameterization.